### PR TITLE
New masks

### PR DESCRIPTION
**Pull request by alsoandanswer**
Adds new sprites for sterile mask and breath masks.
Adds new colorable cloth mask for civillians and dust mask mostly for industrial jobs.

Masks have been move to their own tab in loadout - namely because they have an entire seperate file for them, but apparently haven't been coded in their own tab. Also, they should equip properly on spawn now.
dreammaker_5v5GcrWQaq

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/defines/mutable_appearance.dm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Mutable appearances are an inbuilt byond datastructure. Read the documentation on them by hitting F1 in DM.
+// Basically use them instead of images for overlays/underlays and when changing an object's appearance if you're doing so with any regularity.
+// Unless you need the overlay/underlay to have a different direction than the base object. Then you have to use an image due to a bug.
+
+// Mutable appearances are children of images, just so you know.
+
+/mutable_appearance/New()
+	..()
+	plane = FLOAT_PLANE // No clue why this is 0 by default yet images are on FLOAT_PLANE
```
Comment (Alberyk) [inline code/defines/mutable_appearance.dm]: This is used in other places in game already, stuff that uses this should be tested.

Comment (alsoandanswer) [reply code/defines/mutable_appearance.dm]: Tested, works as normal.

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**